packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm/DataAccessRequestAccessorsFilesForm.test.tsx

```diff
@@ -141,18 +141,13 @@ const defaultProps: DataAccessRequestAccessorsFilesFormProps = {
   onSubmissionCreated: mockOnSubmissionCreated,
 }
 
-async function renderComponent(
-  props: DataAccessRequestAccessorsFilesFormProps,
-) {
-  let renderResult
-  // We must await asynchronous events for our assertions to pass
-  // eslint-disable-next-line @typescript-eslint/require-await
-  await act(async () => {
-    renderResult = render(<DataAccessRequestAccessorsFilesForm {...props} />, {
-      wrapper: createWrapper(),
-    })
+function renderComponent(props: DataAccessRequestAccessorsFilesFormProps) {
+  const user = userEvent.setup()
+  const component = render(<DataAccessRequestAccessorsFilesForm {...props} />, {
+    wrapper: createWrapper({ withErrorBoundary: true }),
```

Comment:
Wrap the component in an error boundary, then we'll verify that the error boundary handled the error. This is much easier to verify compared to checking that the act of rendering the component eventually throws an error.